requirements.txt

```diff
@@ -0,0 +1,189 @@
+#
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
The Integrations template works with Python 3.8, and that's the version in the [Dockerfile](https://github.com/PADAS/gundi-integration-action-runner/blob/602756b7902a86b505c8d94f6d6797b46387a023/docker/Dockerfile#L1). As we are using some features of python 3.10 here, maybe we can upgrade to python 3.10 in the template repo.
cc @chrisdoehring

Reply:
Thanks @marianobrc Good idea.